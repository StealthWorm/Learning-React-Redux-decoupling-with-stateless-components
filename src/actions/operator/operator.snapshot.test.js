import renderer from 'react-test-renderer';
import { operatorClick } from './index';

describe('operatorClick action', () => {
  it('should have the correct shape', () => {
    const operator = '+';
    const action = operatorClick(operator);
    expect(action).toMatchSnapshot();
  });
});