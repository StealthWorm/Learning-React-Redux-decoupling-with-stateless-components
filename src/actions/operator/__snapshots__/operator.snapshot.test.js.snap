// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`operatorClick action should have the correct shape 1`] = `
Object {
  "payload": Object {
    "CLICKED_OPERATOR_KEY": "+",
  },
  "type": "OPERATOR_CLICK_ACTION",
}
`;
